waymo_raw_data_to_argoverse.py

```diff
@@ -216,19 +258,89 @@ def dump_pose(city_SE3_egovehicle, timestamp, log_id):
 		'rotation': [w, x, y, z],
 		'translation': [x,y,z]
 	}
-	json_fpath = f'TEST_RAW_DATA/pose_logs/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
+	json_fpath = f'{parent_path}/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
 	check_mkdir(str(Path(json_fpath).parent))
 	save_json_dict(json_fpath, pose_dict)
 
+def dump_point_cloud(points, timestamp, log_id, parent_path):
```

Comment:
thanks for adding this. do you mind adding a 1-line docstring and type hints here?

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/8931614c713c462a1efbbc0f06a366d44a28df25)